infra/base-images/base-runner/test_all.py

```diff
@@ -0,0 +1,176 @@
+#!/usr/bin/env python3
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+"""Does bad_build_check on all fuzz targets in $OUT."""
+
+import multiprocessing
+import os
+import re
+import shutil
+import subprocess
+import stat
+import sys
+
+TMP_FUZZER_DIR = '/tmp/not-out'
+
+EXECUTABLE = stat.S_IEXEC | stat.S_IXGRP | stat.S_IXOTH
+
+
+def recreate_directory(directory):
+  """Creates |directory|. If it already exists than deletes it first before
+  creating."""
+  if os.path.exists(directory):
+    shutil.rmtree(directory)
+  os.mkdir(directory)
+
+
+def move_directory_contents(src_directory, dst_directory):
+  """Moves contents of |src_directory| to |dst_directory|."""
+  src_files = os.listdir(src_directory)
+  for filename in src_files:
+    src_path = os.path.join(src_directory, filename)
+    shutil.move(src_path, dst_directory)
+
+
+def find_fuzz_targets(directory):
+  """Returns paths to fuzz targets in |directory|."""
+  fuzzing_language = os.getenv('FUZZING_LANGUAGE')
+  fuzz_targets = []
+  for filename in os.listdir(directory):
+    path = os.path.join(directory, filename)
+    if filename.startswith('afl-'):
+      continue
+    if not os.path.isfile(path):
+      continue
+    if not os.stat(path).st_mode & EXECUTABLE:
+      continue
+    with open(path, 'rb') as file_handle:
+      binary = file_handle.read()
+      if b'LLVMFuzzerTestOneInput' not in binary:
+        continue
+      if fuzzing_language != 'python' and b'ELF' not in binary:
```

Comment:
This looks wrong
file "$FUZZER_BINARY" | grep -v ELF
i think we checkout output of file command.

Reply:
Done to match existing behavior, but I always found it weird that we did this instead of reading the first 4 bytes for 0x74ELF (as every ELF needs).